ui/src/components/Dashboard/ResourceTable.js

```diff
@@ -54,13 +56,24 @@ const ResourceTable = ({
   const [headers, setHeaders] = useState([]);
   const [errorMessage, setErrorMessage] = useState(false);
   const [hasError, setHasError] = useState(false);
-
   const classes = useStyles();
+  // eslint-disable-next-line
+  const [tableFilters, setTableFilters] = useTableFilters({});
```

Comment:
Where do you use tableFilters ? (I only see you use setTableFilters)

Reply:
fixed